### DISCUSSION TITLE
[Unknown Mod][Heroes] Adventure map, Automatic capture of empty boat adjacent to the hero sprite

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#9072] [Ideas]
When a hero moves and is positioned in a square adjacent to an empty boat, this empty boat is considered to belong to the player controlling the hero.
This avoids the micromanagement of having to move the hero up (and down) the boat to control it.

This is an option:
"World: Hero automatically captures boat adjacent to its position"

**Comment by oleg-derevenetz** [comment]
The point is that you have to spend additional time (at least a whole day and maybe significantly more) to control a boat, and this is by design. This is not a some kind of "micromanagement", you have to choose between securing a boat and spending the same amount of time to perform another tasks (and a whole day is a lot of time), while just moving on a tile near a boat costs much cheaper in terms of in-game time. This is a breaking change in the game mechanics in comparison to the OG and not just a some minor improvement "for convenience".

**Reply by LeHerosInconnu** [reply]
Hello @oleg-derevenetz,

> The point is that you have to spend additional time (at least a whole day and maybe significantly more) to control a boat, and this is by design. This is not a some kind of "micromanagement", you have to choose between securing a boat and spending the same amount of time to perform another tasks (and a whole day is a lot of time), while just moving on a tile near a boat costs much cheaper in terms of in-game time. This is a breaking change in the game mechanics in comparison to the OG and not just a some minor improvement "for convenience".

I know this is a change from the original game design, which is why this is intended to be an option in my mod, which is aimed at "being primarily designed to have fun, reduce micromanagement and frustration, for single-player game mode". :)
